lib/locale/pl/strings.php

```diff
@@ -3175,5 +3175,6 @@
 $_LANG['customer contacts: '] = 'numery telefonów klienta:';
 
 $_LANG['All users'] = 'Wszyscy użytkownicy';
+$_LANG['Set GPS coordinates'] = 'Ustaw współrzędne GPS';
```

Comment:
Można nazwać to **Determine GPS coordinates automatically** (Wyznacz automatycznie współrzędne GPS)?